features/src/main/feature/camel-features.xml

```diff
@@ -1237,6 +1244,16 @@
     <feature name="camel-jms" version="${project.version}" start-level="50">
         <feature version='${camel.osgi.version.range}'>camel-core</feature>
         <feature version="${camel.osgi.spring.version}">spring-jms</feature>
+        <!-- The SMX bundle of Spring JMS doesn't contain micrometer as part of its import packages, see https://issues.apache.org/jira/browse/SM-5706,
+            so we need to override the imports by wrapping the bundle until it is fixed in the SMX bundle project and in Karaf.
+            The Bundle-Version is updated by Bundle-Version=${spring-version}.2 so that the updated bundle is used
+            instead of the one provided by Karaf.
+        -->
+        <!-- Workaround START -->
+        <bundle start-level="30">wrap:mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-jms/${spring-version}_1$overwrite=merge&amp;Import-package=io.micrometer;resolution:=optional,*;resolution:=optional&amp;Bundle-Version=${spring-version}.2</bundle>
```

Comment:
That will be fix in the next SMX bundles releases (I will prepare the release during the week end).